leshan-server-cf/src/main/java/org/eclipse/leshan/server/californium/impl/LwM2mResponseBuilder.java

```diff
@@ -352,7 +352,7 @@ private LwM2mNode decodeCoapResponse(final LwM2mPath path, final Response coapRe
         LwM2mNode content;
         try {
             // get content format
-            ContentFormat contentFormat = null;
+            ContentFormat contentFormat = ContentFormat.TEXT;
```

Comment:
I have tested other formats like OPAQUE and TLV. But it seems that TEXT has the most chance to decode successfully.